pkg/tms/tmsUtils.go

```diff
@@ -0,0 +1,347 @@
+package tms
```

Comment:
No tests? Or indirect tests, how is the test coverage? Would it make sense to write comprehensive tests for these functions, to be sure they behave as expected? (Maybe not the short ones such as UnmarshalServiceKey, but the more complex ones such as FormNodeIdExtDescriptorMappingWithValidation)

Reply:
The tmsUtils functions will be tested by tests in cmd.
Current test coverage:
FormNodeIdExtDescriptorMappingWithValidation	76.9%
GetYamlAsMap								87.5%
UploadDescriptors								68.5%

Reply:
Added one tmsUtils test which is almost 1:1 copy of the "happy path: 1. get nodes 2. get MTA ext descriptor -> nothing obtained 3. upload MTA ext descriptor to node 4. upload file 5. upload file to node" test from tmsUpload_test. IMHO it doesn't make sense.

Reply:
Then let it. We test it through the cmd, that is fine.

Reply:
OK, I removed tmsUtils_test